modules/nf-core/anndata/barcodes/meta.yml

```diff
@@ -44,11 +48,15 @@ output:
           description: |
             AnnData file containing all cells with barcodes that match those in the input barcodes csv file
           pattern: "*.h5ad"
+          ontologies:
+            - edam: "http://edamontology.org/format_3590" # HDF5
```

Comment:
same here

Reply:
Same answer, to be kept or not accordingly